charts/operator-wandb/charts/app/templates/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.global.customCACerts }}
+{{- if and (not .Values.global.configMapName) .Values.global.customCACerts }}
```

Comment:
I think the chart should probably support both a user supplied config map and individual certs passed at the same time.  Theres no reason to force one or the other